src/draw/drawLineOnIjs.ts

```diff
@@ -42,37 +44,39 @@ export function drawLineOnIjs(
   if (from.column === to.column && from.row === to.row) {
     return newImage;
   }
-  let { row, column } = from;
-  const { row: rowEnd, column: columnEnd } = to;
-
-  const dColumn = Math.abs(columnEnd - column);
-  const columnIncrement = column < columnEnd ? 1 : -1;
-  const dRow = -Math.abs(rowEnd - row);
-  const rowIncrement = row < rowEnd ? 1 : -1;
-  let err = dColumn + dRow;
-  let drawing = true;
-  let e2;
 
-  while (drawing) {
-    newImage.setPixel(column, row, color);
-    e2 = 2 * err;
-    if (e2 >= dRow) {
-      if (column === columnEnd) {
-        drawing = false;
-      } else {
-        err += dRow;
-        column += columnIncrement;
-      }
-    }
-    if (e2 <= dColumn) {
-      if (row === rowEnd) {
-        drawing = false;
-      } else {
-        err += dColumn;
-        row += rowIncrement;
-      }
-    }
+  const numberChannels = Math.min(newImage.channels, color.length);
+  if (
+    [ImageColorModel.GREYA, ImageColorModel.RGBA].includes(image.colorModel)
+  ) {
+    lineAA(
```

Comment:
Anti-aliasing will be an option in the fututre but we don't want it for now

Reply:
ok removed